content/docs/catalog/repositories/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Repositories in the Catalog
+publishedDate: '2024-03-19T14:49:47.0Z'
+description: How repositories are modelled and ingested into the catalog
+---
+
+![Repositories Catalog Table](./repositories-catalog.png)
+
+Repositories are modelled in the Roadie Backstage catalog using the Repository Kind.
+
+### Catalog Completeness
+Repositories allow us to effectively measure the completeness of your catalog - that is, whether all the software in your organisation is represented by entities in the catalog.
+
+Once your Repositories are added, Admins will be able to see a graph of catalog completeness in the Administration section of Roadie. 
+
+You will also find built-in completeness Scorecards and checks in Tech Insights and a Data Source to use to build your own Checks and Scorecards from the same data.
+
+![Catalog Completeness Page](./catalog-completeness-graph.png)
+
+### Autodiscovery
+They will be automatically ingested for you from any GitHub integrations you have set up in Roadie.
+
+By default in Roadie this data will be populated for you and refreshed every 12 hours or whenever a change is made to a repository assuming you have [GitHub Autodiscovery](/docs/catalog/location-management/#setting-up-autodiscovery) set up for your GitHub organisations.
+
+### Ingesting from non-GitHub Sources
+You can use the [Roadie Entity Push API](/docs/api/authorization/) to add repositories from other SCMs other than GitHub via a script.
+
+This would look something like the following:
+
+```bash
+curl https://api.roadie.so/api/catalog/roadie-entities/sets/1 \
+  -X POST \
+  -H 'Accept: application/json' \
+  -H "Authorization: bearer ${ROADIE_API_TOKEN}" \
+  -d '{"apiVersion":"roadie.io/v1","kind":"Repository","metadata":{"name":"test1","description":"A description of the repository","scmOwner":"RoadieHQ","defaultBranch":"main","visibility":"private","language":"TypeScript","securityAndAnalysis":{"secretScanning":"disabled","secretScanningPushProtection":"disabled","secretScanningValidityChecks":"disabled","dependabotSecurityUpdates":"enabled"},"lastPush":"2024-01-04","createdAt":"2022-01-21","potentialOwners":["group:default/somegroup"],"isFork":"false","commits":{"lastWeek":0,"lastMonth":12,"lastThreeMonths":12,"lastSixMonths":14,"lastYear":23}},"spec":{"hasPart":["component:default/authx"]}}'
+```
+
+### Schema
+Repositories follow a unique schema that has a lot of metadata fields designed to help identify what repositories are active or important, who owns the repository, and what entities are connected to it.
+
+NB: They use a new `apiVersion` of `roadie.io/v1` and `Kind` of `Repository`.
+
+```json
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "RepositoryEntityV1",
+  "description": "A repository represents an SCM repository and its related entities.",
+  "examples": [
```

Comment:
I know this say examples and should be used but maybe worth adding a copy-pastable repo entity as a standalone thing in this (or yaml) format also

Reply:
Got one in the sample curl already. I think we only accept json on the entity push endpoints